tests/PHPStan/Analyser/data/sscanf.php

```diff
@@ -37,8 +37,14 @@ function fscanfFormatInference($r) {
 	assertType('int|null', $year);
 }
 
-function fooo() {
+function fooo(string $s) {
+	// %0s returns the whole string
+	assertType('array{non-empty-string|null}|null', sscanf( "0" , "%0s"));
 	assertType('array{non-empty-string|null}|null', sscanf( "123456" , "%0s"));
```

Comment:
actually for a `non-falsy-string` $string this test could lead to a more precise `'array{non-falsy-string|null}|null'`, but I feel like `%0s` as a format is more or less a theoretical thing.. never seen it used anywhere.. I think people would use `%s` instead